src/meta/node/src/lib.rs

```diff
@@ -196,7 +198,10 @@ pub fn start(opts: MetaNodeOpts) -> Pin<Box<dyn Future<Output = ()> + Send>> {
                     .map(|x| x.to_string())
                     .collect(),
                 credentials: match opts.etcd_auth {
-                    true => Some((opts.etcd_username, opts.etcd_password)),
+                    true => Some((
+                        opts.etcd_username,
+                        opts.etcd_password.expose_secret().to_string(),
```

Comment:
Hmmm, I feel ***ideally*** we should make all the intermediate stuff `Secret`, and reveal secret until we have to. Just checked `MetaStoreBackend` also `#[derive(Debug)]`. 🤣

Just raise this point. Not a hard requirement for now.

Reply:
Good point.

Reply:
https://github.com/risingwavelabs/risingwave/issues/12673#issuecomment-1780729326